core/src/main/java/bio/terra/pearl/core/service/export/TsvExporter.java

```diff
@@ -21,42 +22,22 @@ public TsvExporter(List<ModuleFormatter> moduleExportInfos, List<Map<String, Str
      * can be supported
      */
     @Override
-    public void export(OutputStream os) {
-        PrintWriter printWriter = new PrintWriter(os);
+    public void export(OutputStream os) throws IOException {
```

Comment:
I don't think this method should throw a checked exception for IOException -- there's nothing the caller can do.  Let's handle the exception here, and turn it into an IOInternalException